libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -3604,6 +3610,7 @@ export const findCryptoCurrencyByManagerAppName = (
 export const hasCryptoCurrencyId = (id: string): boolean =>
   id in cryptocurrenciesById;
 
+// TODO: signature should be getCryptoCurrencyById(id: CryptoCurrencyId)
```

Comment:
What's preventing this, at the moment?

Reply:
It's impacting a lot of code way outside of the scope of this PR